### DISCUSSION TITLE
IBAX'a ilgi çok yüksek.

### DISCUSSION THREAD

**Discussion by blastaway07** [IBAX-io/go-ibax#251] [Bounty Hunter]
IBAX'i hemen hemen tüm platformlar da görüyorum, AVAX gibi olabilir mi buna emin değilim ama umarım olur.
Kolay gelsin :)

**Comment by ankateam41** [comment]
geleceğe inanıyorum:)

**Comment by byblckd** [comment]
aslında çok ilgi yok sadece Türkiye kullanıcların ilgisi yüksek olduğundan sizler sosyal medyada öyle görüyorsunuz. Avrupa ve Amerika merkezli sosyal medyalarda pek ilgi çekmiyor şuanlık biraz daha elle tutulur şeyler yaptıklarında o zaman ilgi çekmeye başlayabilir
